### DISCUSSION TITLE
Configure TP-Link TL-SG108E Switch

### DISCUSSION THREAD

**Discussion by nicholaswkc34** [Security-Onion-Solutions/securityonion#11780] [2.4] [answered]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

other (please provide detail below)

### Hardware Specs

Meets minimum requirements

### CPU

Quad Core

### RAM

20GB

### Storage for /

300GB

### Storage for /nsm

300GB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Dear all forumers, I had installed SO 2.4.20 and completed the so-setup but I didn't find the setup for whether to use TUN/Span port selection at so-setup. 

Another questions is I had configured router TP-Link TL-SG 108E to mirror on port 8 but the link is down. How to enable it and let it capute the inbound traffic? There is no ip address associated with it yet. 

Please help. Thanks. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by Acewiza** [comment]
Monitor port on the Onion doesn't care what you connect it to.  But it needs  link.  Might be a config issue on that port-8.

**Comment by nicholaswkc34** [comment]
Derar all forumers, I just enable the port mirroring feature on port 8 for my TP-Link TL-SG108E switches. 

![javaw_Umrz5nOnMH](https://github.com/Security-Onion-Solutions/securityonion/assets/44965697/1dc56feb-e131-409e-9c1f-50397a2a276f)

This is the screenshot of the switches port mirroring.

**Reply by Acewiza** [reply]
Yeah sorry, but the only TP-Link device I own is a DSL router, so I'm out.

**Comment by nicholaswkc34** [comment]
Dear all forumers, Please help.

**Comment by reyesj2** [comment, marked as answer]
Looks like you need to 'enable' ingress / egress on each port you want to mirror traffic out to port 8.

While you troubleshoot your switch you can use the influxdb dashboard to monitor ingested traffic speed and give you insight into when your Security Onion box starts seeing traffic. https://docs.securityonion.net/en/2.4/influxdb.html?highlight=influxdb#influxdb

**Comment by nicholaswkc34** [comment]
Good info here.
